### ISSUE TITLE
Unable to close drawer with keyboard

### ISSUE THREAD

**Issue by zdavis** [ManifoldScholar/manifold#2271]
Focus is on the close button, but I can't seem to close it. This was recorded in your focus branch.

![image](https://user-images.githubusercontent.com/255909/61158767-d23d2c80-a4ae-11e9-90cc-60edfd6f7e28.png)
